var/spack/repos/builtin/packages/py-brian2/package.py

```diff
@@ -22,6 +22,9 @@ class PyBrian2(PythonPackage):
     depends_on("python@2.7:", type=("build", "run"))
     depends_on("python@3.6:", type=("build", "run"), when="@2.4:")
     depends_on("python@3.7:", type=("build", "run"), when="@2.5:")
+    # in newer pip versions --install-options does not exist
```

Comment:
Technically the flag name is `--install-option` singular

Reply:
I called it `--install-options` because its called like that in the [Changelog](https://pip.pypa.io/en/stable/news/#v23-1) and the [Issue](https://github.com/pypa/pip/issues/11358). This is changed now.